comp/core/workloadmeta/collectors/internal/containerd/container_builder_test.go

```diff
@@ -9,6 +9,8 @@ package containerd
 
 import (
 	"context"
+	"fmt"
+	"github.com/containerd/containerd/cio"
```

Comment:
could you move the `github.com/containerd/containerd/cio` outside the golang official package